api/tasks/clean_document_task.py

```diff
@@ -39,8 +42,17 @@ def clean_document_task(document_id: str, dataset_id: str, doc_form: str):
                 db.session.delete(segment)
 
             db.session.commit()
-            end_at = time.perf_counter()
-            logging.info(
-                click.style('Cleaned document when document deleted: {} latency: {}'.format(document_id, end_at - start_at), fg='green'))
+        if file_id:
+            file = db.session.query(UploadFile).filter(
+                UploadFile.id == file_id
+            ).first()
+            if file:
+                storage.delete(file.key)
```

Comment:
It is recommended to add a try-raise so that the main process is not interrupted if a file is deleted abnormally.